adal/src/main/java/com/microsoft/aad/adal/StorageHelper.java

```diff
@@ -479,8 +475,6 @@ private synchronized boolean doesKeyPairExist() throws GeneralSecurityException,
             // java.lang.NullPointerException: Attempt to invoke interface method 
             // 'int android.security.IKeystoreService.exist(java.lang.String, int)' on a null object reference
             // To avoid app from crashing, re-throw as checked exception
-            ClientAnalytics.logEvent(new AndroidKeyStoreFailureEvent(
-                    new InstrumentationPropertiesBuilder(exception)));
```

Comment:
Not yet.

---
In reply to: [90366816](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90366816) [](ancestors = 90366816)